install_dependencies.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
If this file needs bash capabilities, it should be renamed to `install_dependencies.bash`. Otherwise, the shebang should be changed to `#!/bin/sh`.

Reply:
Good catch. Copied this from carma-time-lib where it probably also needs to be fixed. I would guess this is a copy paste mistake that is also present in carma-builds.